etl/src/main/java/org/pih/hivmigration/etl/sql/util/FileParser.java

```diff
@@ -0,0 +1,32 @@
+package org.pih.hivmigration.etl.sql.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FileParser {
+
+    // TODO confirm that this works with CSVs with more than one column (if we ever need it)
```

Comment:
Yeah, bad practice to not even test at all, but from a quick review we may never need this beyond the zl-identifier.csv, which is basically one column

Reply:
Fine, but might be worth just pulling in an existing tried and tested CSV library that does this out of the box - something like opencsv, etc.

Reply:
True.. I was considering that, but this seemed simple enough to write... worth exploring further if there are other csv files we end up needing to import.